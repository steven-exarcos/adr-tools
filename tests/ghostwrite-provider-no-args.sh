adr init
if adr ghostwrite gemini
then
    echo ERROR: should have failed
fi