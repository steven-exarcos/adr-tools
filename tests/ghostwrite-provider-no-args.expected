adr init
doc/adr/0001-record-architecture-decisions.md
if adr ghostwrite gemini
then
    echo ERROR: should have failed
fiError: Please specify an ADR file or number.
Usage: adr ghostwrite [provider] <adr-file-or-number>
Supported providers: gemini (default), claude, chatgpt
